static/_redirects

```diff
@@ -13,3 +13,7 @@
 /platform/23.2.0/*  /platform/23.2/:splat   301
 /platform/23.1.0/*  /platform/23.1/:splat   301
 /platform/22.4.0/*  /platform/22.4/:splat   301
+
+# Redirect from old FAQ to new FAQ
+
+/platform/24.1/faq/*  /platform/24.1/troubleshooting_and_faqs/troubleshooting/:splat   301
```

Comment:
Sorry, missed this earlier. Does this pick up the faqs.mdx page?

Reply:
Good catch, thank you!